apps/src/blockly/addons/cdoPathObjectZelos.js

```diff
@@ -0,0 +1,15 @@
+import GoogleBlockly from 'blockly/core';
+
+export default class CdoPathObject extends GoogleBlockly.zelos.PathObject {
```

Comment:
for my learning: what is `zelos`?

Reply:
Learned this recently, they're different blockly renderers. I think their docs are kind of broken at the moment so I can't see the main [zelos renderer page](https://developers.google.com/blockly/reference/js/Blockly.zelos.Renderer). No idea why they're using greek names 🤦